backend/src/model/dto.rs

```diff
@@ -215,9 +216,15 @@ pub struct MapDto {
     pub location: Option<Coordinates>,
     /// The id of the owner of the map.
     pub owner_id: Uuid,
+    /// The geometry of the map.
+    ///
+    /// E.g. `{"rings": [[{"x": 0.0,"y": 0.0},{"x": 1000.0,"y": 0.0},{"x": 1000.0,"y": 1000.0},{"x": 0.0,"y": 1000.0},{"x": 0.0,"y": 0.0}]],"srid": 4326}`
```

Comment:
Please describe the geometry. Are there any required properties of that geometry? (E.g. closed)

Also describe how it looks serialized (maybe link to where it is described), e.g. what is "rings"?

Probably the description better should be in the endpoint to not duplicate the information.

Reply:
> Please describe the geometry. Are there any required properties of that geometry? (E.g. closed)

As described above I actually don't know why it looks like this.
I think it has to be closed, so the last coordinate has to be the same as the first one, but I'm not sure.

> Also describe how it looks serialized (maybe link to where it is described), e.g. what is "rings"?

I don't know exactly what this means. The serialized struct is `Polygon<Point>`, but I don't think that's what you mean. The fields in Rust look pretty much the same as the JSON.

> Probably the description better should be in the endpoint to not duplicate the information.

I think it fits better here. The DTOs get displayed in Swagger as well, and that way it fits to the struct.